### DISCUSSION TITLE
How does it work?

### DISCUSSION THREAD

**Discussion by Metsi103** [KIvoy/nikke-assistant#14] [Q&A]
Hello there, I'd like to use the bot, so I downloaded the zip folder and unzipped. The only problem is: I can't find the Nikke_Assistant.exe file. I only got a .ini file that opens a note. What should I do to figure out how does it work?

**Comment by KIvoy** [comment]
Hi @Metsi103 , not sure if you have this one figured out already, but the application file you should be running is called "nikke-assistant". You should be able to find it under the root directory of the zip you've downloaded.
